### ISSUE TITLE
Problem running the Go example with sphinx v2.1.2

### ISSUE THREAD

**Issue by ArmandBENETEAU** [readthedocs/sphinx-autoapi#168]
Dear all,

We would like to use your sphinx-autoapi tool to auto-document our Go code. It would be very useful for our company since we would use the Sphinx to document our python code and our Go code.

I have Sphinx v2.1.2 running on my Ubuntu 18.04.2 LTS laptop, and it works really well to document python 3.6 code. 

Therefore, I tried to creating the documentation of your Go example code by doing the following procedure : 

1/ Getting your example from your GIT repository

2/ Creating the following repository structure and getting the Sphinx Makefile : 
.
├── build
├── Makefile
└── source
........├── conf.py
........├── example
..................└── godocjson.go
........├── index.rst
........├── _static
........└── _templates

Sphinx Makefile : 

``` make
# Minimal makefile for Sphinx documentation
#

# You can set these variables from the command line, and also
# from the environment for the first two.
SPHINXOPTS    ?=
SPHINXBUILD   ?= sphinx-build
SOURCEDIR     = source
BUILDDIR      = build

# Put it first so that "make" without argument is like "make help".
help:
	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)

.PHONY: help Makefile

# Catch-all target: route all unknown targets to Sphinx using the new
# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
%: Makefile
	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```

Unfortunately, when I run the command `make html`, a lot of warnings and an exception are appearing. This result in no documentation being created. You can find here below the complete logs. I would like to know what is happening, is it possible to resolve it and if yes, how I can resolve this exception?

``` bash
Sphinx v2.1.2 en cours d'exécution
création du répertoire de sortie... fait
[AutoAPI] Loading Data
[AutoAPI] Mapping Data
[AutoAPI] Rendering Data
construction en cours [mo]:targets for 0 po files that are out of date
construction en cours[html]: targets for 1 source files that are out of date
updating environment: 3 added, 0 changed, 0 removed
lecture des sources...[100%] index                                                                                                                                                                                 
/home/armand/Bureau/Test/source/autoapi/main/index.rst:21: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/home/armand/Bureau/Test/source/autoapi/main/index.rst:48: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/home/armand/Bureau/Test/source/autoapi/main/index.rst:77: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/home/armand/Bureau/Test/source/autoapi/main/index.rst:2: WARNING: 4 column based index found. It might be a bug of extensions you use: [('single', 'main (package)', 'package-main', 'main')]
/home/armand/Bureau/Test/source/autoapi/main/index.rst:: WARNING: 4 column based index found. It might be a bug of extensions you use: [('single', 'main.Func (Golang type)', 'main.Func', 'main.Func')]
/home/armand/Bureau/Test/source/autoapi/main/index.rst:30: WARNING: 4 column based index found. It might be a bug of extensions you use: [('single', 'main.CopyFuncs (Golang function)', 'main.CopyFuncs', 'main.CopyFuncs'), ('single', 'None (Golang function)', None, None)]
/home/armand/Bureau/Test/source/autoapi/main/index.rst:: WARNING: 4 column based index found. It might be a bug of extensions you use: [('single', 'main.Note (Golang type)', 'main.Note', 'main.Note')]
/home/armand/Bureau/Test/source/autoapi/main/index.rst:: WARNING: 4 column based index found. It might be a bug of extensions you use: [('single', 'main.Package (Golang type)', 'main.Package', 'main.Package')]
/home/armand/Bureau/Test/source/autoapi/main/index.rst:57: WARNING: 4 column based index found. It might be a bug of extensions you use: [('single', 'main.CopyPackage (Golang function)', 'main.CopyPackage', 'main.CopyPackage'), ('single', 'None (Golang function)', None, None)]
/home/armand/Bureau/Test/source/autoapi/main/index.rst:: WARNING: 4 column based index found. It might be a bug of extensions you use: [('single', 'main.Type (Golang type)', 'main.Type', 'main.Type')]
/home/armand/Bureau/Test/source/autoapi/main/index.rst:: WARNING: 4 column based index found. It might be a bug of extensions you use: [('single', 'main.Value (Golang type)', 'main.Value', 'main.Value')]
/home/armand/Bureau/Test/source/autoapi/main/index.rst:85: WARNING: 4 column based index found. It might be a bug of extensions you use: [('single', 'main.CopyValues (Golang function)', 'main.CopyValues', 'main.CopyValues'), ('single', 'None (Golang function)', None, None)]
recherche des fichiers périmésaucun résultat
pickling environment... fait
checking consistency... fait
document en préparation... fait
écriture de la sortie...[ 66%] autoapi/main/index                                                                                                                                                                  
une exception s'est produit :
  File "/home/armand/.local/lib/python3.6/site-packages/docutils/writers/_html_base.py", line 371, in starttag
    assert value is not None
AssertionError
The full traceback has been saved in /tmp/sphinx-err-hdc8odti.log, if you want to report the issue to the developers.
Merci de rapporter ceci s'il s'agit d'une erreur utilisateur, afin d'améliorer le message d'erreur à l'avenir.
Un rapport d'erreur peut être déposé dans le système de tickets à <https://github.com/sphinx-doc/sphinx/issues>. Merci !
Makefile:20: recipe for target 'html' failed
make: *** [html] Error 2
```

Thank you by advance !

**Comment by AWhetter** [comment]
The golang support does not have a maintainer to give it the attention it needs. It's still considered an Alpha part of autoapi. If you manage to fix this issue then we would be grateful for a pull request.

**Comment by AWhetter** [comment]
Closed in #172
